lib/TemperatureCalc.js

```diff
@@ -0,0 +1,18 @@
+class TemperatureCalc{
+  constructor(temperature, type){
+    this.temperature = temperature;
+    this.type = type
+  }
+
+  calculator(){
```

Comment:
Not particular, adopted this format so I can have a re-useable instance of the variable across multiple functions for reason of scalable, but I guess that doesn't apply in this case since this is just a small class. Let me try to pass it in the function, thanks